ee/cis/macos-13/cis-policy-queries.yml

```diff
@@ -2105,6 +2105,44 @@ spec:
 ---
 apiVersion: v1
 kind: policy
+spec:
+  name: CIS - Audit Safari Web Browser History and Remove History Items (organisation decision needed)
```

Comment:
Nit: `organization` (though organisation is valid, we use this version of the word in the repository):

```
rg -i organization | wc -l
     662
rg -i organisation | wc -l
       0
```